images/vertical-pod-autoscaler/main.tf

```diff
@@ -0,0 +1,71 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+locals {
+  components = toset(["admission-controller", "recommender", "updater"])
+}
+
+variable "target_repository" {
+  description = "The docker repo into which the image and attestations should be published."
+}
+
+module "latest" {
+  for_each = local.components
+  source   = "../../tflib/publisher"
+
+  target_repository = "${var.target_repository}-${each.key}"
+  config            = file("${path.module}/configs/latest.${each.key}.apko.yaml")
+}
+
+module "dev" {
+  for_each = local.components
```

Comment:
We don't need to foreach this, it's a constant

Reply:
Good catch, I'll fix this in cert-manager too.